### ISSUE TITLE
Ubuntu MATE: Not in window list, no window decoration, window always on top

### ISSUE THREAD

**Issue by killerkalamari** [evolus/pencil#634]
* Operating system: Ubuntu MATE 18.04
* Pencil version: 3.1.0.ga

There are major problems with the Pencil GUI on Ubuntu MATE:
* The window does not appear on the window list (taskbar)
* There is no window decoration to move, minimize, maximize, close, or resize the window
* The window is always on top of all other windows

![Screenshot at 2020-07-14 09:15:05](https://user-images.githubusercontent.com/5439104/87450011-cc42c580-c5b2-11ea-9f2b-40cd0d8876a5.png)

**Comment by hydexon** [comment]
Using Ubuntu Mate 20.04 LTS, same here.
One possible workaround is run Pencil with a terminal will work normally.

**Comment by spider4216** [comment]
The same issue on Linux Mint 20 Ulyana MATE

**Comment by spider4216** [comment]
> Using Ubuntu Mate 20.04 LTS, same here.
> One possible workaround is run Pencil with a terminal will work normally.

Confirm, if run via terminal - everything is okay

**Comment by jkpubsrc** [comment]
No, if run via terminal, things are not okay. In that case the application runs in a small window. If I specify a command line argument, the application is started at fullscreen. Then I have a window decoration (but the application complains about the command line argument not making any sense).

So my guess is: This is an issue with how the main window is opened in the first place.

Side effect: The application does not get the focus. What ever keys I press, the key presses still are directed to the terminal window, not the application (which **is** running in the foreground).

Solution: 

As this is a **show stopper* I had a look at the sources. But I'm not familiar with Electron and not familiar with the application. I could not locate the code that must be modified in order to fix this.

Again: This is a **show stopper**.

**Comment by Jeeppler** [comment]
@killerkalamari the command line argument trick did not work for me.

**Comment by killerkalamari** [comment]
> @killerkalamari the command line argument trick did not work for me.

I think you may have meant to tag someone else. I'm just the one who opened the bug. I gave up on being able to use this program a long time ago.

**Comment by plonibarploni** [comment]
> I gave up on being able to use this program a long time ago

@killerkalamari Have you found a different program to replace it?

**Comment by killerkalamari** [comment]
No, not really. IIRC, I needed to make 2 diagrams. I used LibreOffice Draw for one and dia for the other.

**Comment by Jeeppler** [comment]
Actually, I resized the window of my VirtualBox VM (made it smaller), that worked.

**Comment by acuerol** [comment]
Hi, I solved it commenting the line 78 (`mainWindow.hide();`) in the file `./index.js`
